src/test/javascript/spec/component/exam/manage/exam-management.service.spec.ts

```diff
@@ -404,4 +527,47 @@ describe('Exam Management Service Tests', () => {
 
         httpMock.expectOne({ method: 'POST', url: `${service.resourceUrl}/${course.id!}/exams/${mockExam.id!}/register-course-students` });
     });
+
+    it('should find all locked submissions from exam', fakeAsync(() => {
+        // GIVEN
+        const mockExam: Exam = { id: 1 };
+        const mockResponse = [new TextSubmission()];
+        const expected = [new TextSubmission()];
```

Comment:
It does look odd to me to compare two different objects for equality, but doesn't seem to hurt in this case.
(Usually I prefer the `{ id: 1 } as TextSubmission` like syntax, but you don't need to change that here imho.)